test/test_jit.py

```diff
@@ -567,6 +567,42 @@ def __init__(self):
 
 
 class TestJit(JitTestCase):
+    @unittest.skipIf(not RUN_CUDA, "requires CUDA")
+    def test_large_nbr_kernel_args(self):
+        class Recurrence(nn.Module):
+            def __init__(self, input_size, seq_len):
+                super(Recurrence, self).__init__()
+                self.input_size = input_size
+                self.batch_first = True
+                self.seq_len = seq_len
+
+            def forward(self, input):
+                if self.batch_first:
+                    input = input.transpose(0, 1)
+
+                # Main loop
+                output = []
+                for i in range(self.seq_len):
```

Comment:
Using input.size(0) leads to the following warning. Since this is a test, it should be ok. Change done.
test_large_nbr_kernel_args (__main__.TestJit) ... test/test_jit.py:581: TracerWarning: Converting a tensor to a Python index might cause the trace to be incorrect. We can't record the data flow of Python values, so this value will be treated as a constant in the future. This means that the trace might not generalize to other inputs!
  for i in range(input.size(0)):

Reply:
Put seq_len back as a module attribute to avoid the warning.